src/Compilers/CSharp/Test/Emit/PDB/PDBAsyncTests.cs

```diff
@@ -159,7 +159,7 @@ .maxstack  3
    -IL_00e5:  ldsfld     ""System.Threading.AutoResetEvent Driver.CompletedSignal""
     IL_00ea:  callvirt   ""bool System.Threading.EventWaitHandle.Set()""
     IL_00ef:  pop
-   ~IL_00f0:  leave.s    IL_010c
+    IL_00f0:  leave.s    IL_010c
```

Comment:
Right, and this seems more robust solution since ideally we should put the hidden SP not on the goto, but on the landing/label IL that is not otherwise associated with any code.